orion/sites/admin/src/location/locationEdit.js

```diff
@@ -30,11 +60,44 @@ export class LocationEditPage extends React.Component {
         console.log("delete button clicked");
     }
 
-    handleChange() {
-        console.log("handle change");
+    onClickSave() {
+        let location = {
+            locId: this.state.inputlocId,
+            street: this.state.inputStreet,
+            state1: this.state.state1,
```

Comment:
`state: this.state.inputState`

This means the field is named "state" which is assigned to value `this.state.inputState`.
Same with the below fields